package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "courses-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.12",
+    "@types/react": "^18.0.27",
+    "@types/react-dom": "^18.0.10",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "uuid": "^9.0.0",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "eslint": "eslint --ext .js --ext .jsx src",
+    "eslint:fix": "eslint --fix --ext .js --ext .jsx src",
```

Comment:
have you tried to use these scripts? 
Do they work for you?

Reply:
Yes, it works.

Reply:
Really? 🤔 That's strange. They don't work for me.
And I wonder how can they work for you considering that they are set up to check .js an .jsx files, while you have .ts and .tsx files in your project

Reply:
I am not sure. But it works :). If you want i can change .js and .jsx extensions to .ts and .tsx i tried these are also fine.

Reply:
I have updated to .ts and .tsx